README.md

```diff
@@ -119,3 +119,7 @@ To run [the app we're running remotely](http://daattali.com/shiny/bingo/) on you
 ``` r
 launch()
 ```
+
+<!-- badges: start -->
```

Comment:
I'll fix this when I make some other changes. But this should have gone in README.Rmd, not README.md, FYI.